website/source/docs/agent/options.html.md

```diff
@@ -883,6 +883,74 @@ Consul will not enable TLS for the HTTP API unless the `https` port has been ass
 * <a name="disable_keyring_file"></a><a href="#disable_keyring_file">`disable_keyring_file`</a> - Equivalent to the
   [`-disable-keyring-file` command-line flag](#_disable_keyring_file).
 
+* <a name="gossip_lan"></a><a href="#gossip_lan">`gossip_lan`</a> - (Advanced) This object contains a number of sub-keys
```

Comment:
I'd add:

> Tuning these improperly can cause Consul to fail in unexpected ways.

Or something similarly strong language wise about potentially shooting yourself in the foot.

Reply:
And maybe even bold that line 😄